src/main/java/ca/bc/gov/chefs/etl/forms/pcd/upcc/pcPatientServices/processor/PcdUpccPCPSApiResponseProcessor.java

```diff
@@ -0,0 +1,144 @@
+package ca.bc.gov.chefs.etl.forms.pcd.upcc.pcPatientServices.processor;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import org.apache.camel.Exchange;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+import ca.bc.gov.chefs.etl.constant.Constants;
+import ca.bc.gov.chefs.etl.constant.PCDConstants;
+import ca.bc.gov.chefs.etl.core.model.IModel;
+import ca.bc.gov.chefs.etl.core.model.SuccessResponse;
+import ca.bc.gov.chefs.etl.core.processor.BaseApiResponseProcessor;
+import ca.bc.gov.chefs.etl.forms.pcd.upcc.pcPatientServices.json.Root;
+import ca.bc.gov.chefs.etl.forms.pcd.upcc.pcPatientServices.model.PCPSUpccSubmission;
+import ca.bc.gov.chefs.etl.forms.pcd.upcc.pcPatientServices.model.PCPSUpccSubmissionData;
+import ca.bc.gov.chefs.etl.util.CSVUtil;
+import ca.bc.gov.chefs.etl.util.FileUtil;
+import ca.bc.gov.chefs.etl.util.JsonUtil;
+
+public class PcdUpccPCPSApiResponseProcessor extends BaseApiResponseProcessor {
+
+  @SuppressWarnings("unchecked")
+  @Override
+  public void process(Exchange exchange) throws Exception {
+    String payload = exchange.getIn().getBody(String.class);
+    payload = JsonUtil.roundDigitsNumber(payload);
+    ObjectMapper mapper = new ObjectMapper();
+    List<Root> pcpsModels = mapper.readValue(payload, new TypeReference<List<Root>>() {
+    });
+    List<PCPSUpccSubmission> parsedUpccPCPS = parsePCPSRequest(pcpsModels);
+
+    // validateRecordCount(pcpsModels, parsedUpccPCPS);
```

Comment:
Is there any reason this is commented out?

Reply:
I don't think so, must have been a missed comment, this has now been uncommented.